roles/roles_test.py

```diff
@@ -0,0 +1,29 @@
+"""
+Tests for roles definition
+"""
+from unittest import TestCase
+
+from roles import roles
+from roles.models import Role
+
+# pylint: disable=no-self-use
+
+
+class RolesTest(TestCase):
+    """
+    Tests for Roles definitions
+    """
+
+    def test_staff_role_id(self):
+        """
+        Test for the Staff role ID
+        """
+        assert hasattr(roles.Staff, 'ROLE_ID')
+        assert roles.Staff.ROLE_ID == Role.STAFF
+
+    def test_instructor_role_id(self):
+        """
+        Test for the Staff role ID
```

Comment:
these two test cases can be combined into one pretty easily, but i'll leave that up to you. if you decide not to do that, you should update this docstring 😄